app/views/layouts/template.pug

```diff
@@ -5,7 +5,7 @@ html(lang='en')
     include ../include/head.pug
     link(rel='stylesheet', href='/css/channel.css')
 
-  body.layout-name
+  body(class='layout-name' + rotation)
```

Comment:
If you want to use this format for classname, you need a space after layout-name so the classnames aren't scrunched together into one word. Personally, I think it's more visually appealing/easier to understand at a glance to use the following format:

```pug
body.layout-name(class=rotation)
```

This will always apply the `layout-name` class and conditionally apply the rotation class if it's set.